content/news/051/index.md

```diff
@@ -90,6 +90,35 @@ You can support development by [purchasing Way of Rhea on Steam][wor], or
 [wor-mail]: https://anthropicstudios.com/newsletter/signup/tech
 [wor-cps]: https://www.cerebralpuzzleshowcase.com/
 
+### [Untitled Pixel Wizard Game][pixel-wizards]
+
+[![A target being destroyed by a rudimentary flame thrower spell](untitled-pixel-wizards-game.gif)][pixel-wizards]
```

Comment:
Looks like you forgot to add the gif ; for reminder, try to keep its size low (See [contributing.md](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/source/CONTRIBUTING.md#style-guidelines))

Reply:
:facepalm: woops - fixed! Thanks.